portfolio/src/main/webapp/blogpage.css

```diff
@@ -0,0 +1,118 @@
+body{
```

Comment:
it looks like this file replicates a lot of the logic from your other page. Can we create a shared CSS file for the pole design section, and import that to each page that uses it?